projects/storefrontlib/src/cms-components/myaccount/update-email/update-email.module.ts

```diff
@@ -30,9 +34,9 @@ import { FormErrorsModule } from '../../../shared/index';
         },
       },
     }),
+    UpdateEmailService,
```

Comment:
No need to provide this explicitly. The service is provided in 'root' and that's enough. We like this pattern so that it is more optional.